system-status/system_status.py

```diff
@@ -0,0 +1,104 @@
+import boto3
+import json
+import logging
+import os
+import sqlalchemy
+
+from notifications_utils.system_status import determine_notification_status, determine_site_status, TEMPLATES, THRESHOLDS
+
+URL_API = os.getenv("system_status_api_url")
+URL_ADMIN = os.getenv("system_status_admin_url")
+DB_CONN_STRING = os.getenv("sqlalchemy_database_reader_uri")
+BUCKET_NAME = os.getenv("system_status_bucket_name")
+
+# get list of UUIDS from TEMPLATES
+TEMPLATE_IDS = ",".join(["'{}'".format(x) for x in [template_id for notification_type in TEMPLATES for template_id in TEMPLATES[notification_type].values()]])
+
+
+# Build SQL query
+SQL = ("SELECT      template_id,  AVG(extract(epoch from (n.sent_at - n.created_at)))*1000  as time_to_send "
+        "FROM       notifications n "
+        "WHERE      JOB_ID is null "
+        "AND        template_id in ({}) "
+        "AND        created_at >= NOW() - INTERVAL '5 MINUTES' "
+        "GROUP BY   template_id ".format(TEMPLATE_IDS))
+
+def lambda_handler(event, context):
+    
+    try:
+        # connect to postgres db
+        db = sqlalchemy.create_engine(DB_CONN_STRING, future=True)
+        with db.connect() as conn:
+            dbresults = conn.execute(sqlalchemy.sql.text(SQL))
+            conn.commit()    
+
+    except sqlalchemy.exc.OperationalError as e:
+        logging.error("System-status lambda (db connection): {}".format(e))
+        db.dispose();
```

Comment:
will this throw another error if l#30 wasnt created?

Reply:
Probably - I removed them all now so all good!